### PR TITLE
Update documentation, Add titles to sections

### PR DESCRIPTION
**Pull request by morpheus2448**

### CODE REVIEW

File: podman-compose.dev.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+services:
+  serge:
```
Comment (gaby) [inline podman-compose.dev.yml:2]: Whats the difference between this file and the current one?

File: README.md [hunk 2]
```diff
@@ -107,3 +107,9 @@ git clone https://github.com/serge-chat/serge.git
 cd serge/
 docker compose -f docker-compose.dev.yml up --build
 ```
+Or, for systems with Podman:
```
Comment (gaby) [inline README.md:110]: If the file is the same, compose will be the same

File: web/src/routes/chat/[id]/+page.svelte [hunk 3]
```diff
@@ -282,7 +282,7 @@
   <div class="w-full border-b border-base-content/[.2]">
     <div class="h-8 px-2 md:container md:mx-auto md:px-0">
       <div class="w-full h-full relative flex items-center justify-center">
-        <div class="flex flex-row items-center justify-center color-base-300">
+        <div class="flex flex-row items-center justify-center color-base-300" title="Model">
```
Comment (gaby) [inline web/src/routes/chat/[id]/+page.svelte:285]: Thanks!

File: podman-compose.dev.yml (hunk 1 above)
Comment (morpheus2448) [reply podman-compose.dev.yml:2]: Adding :z at the end of lines 8 and 9 so podman can access the volumes without having to disable SELinux.

Comment (gaby) [reply podman-compose.dev.yml:2]: @morpheus2448 You can add those to the existing compose file and delete this one

Comment (morpheus2448) [reply podman-compose.dev.yml:2]: ok

File: README.md [hunk 4]
```diff
@@ -107,3 +107,9 @@ git clone https://github.com/serge-chat/serge.git
...
+```bash
```
Comment (gaby) [inline README.md:111]: This section can be removed now

**Review by gaby — CHANGES_REQUESTED** [review]

**Comment by gaby** [comment]
@morpheus2448 The web code has to be formatted using `prettier`. 

```bash
cd web/
npm install -g prettier
npm run format
```

**Comment by morpheus2448** [comment]
@gaby I think this PR is OK now?

**Review by gaby — APPROVED** [review]

**Pull request merged**